Gems/ROS2/Code/Source/RobotImporter/Pages/CheckAssetPage.cpp

```diff
@@ -139,4 +149,22 @@ namespace ROS2
         }
     }
 
+    void CheckAssetPage::RefreshTimerElapsed(){
+        for (int i =0; i < assetsId.size(); i++)
+        {
+            const auto &assetId  = assetsId[i];
```

Comment:
style, you could use c++17 style
```
for (const auto& assetId : assetsId)
{ ... }

```
if assetsId is a member, it should probably be m_assetsId?

Reply:
Reabsaed all members.
I need index for Qt row down in the code, so I cannot use a range-base loop.